### ISSUE TITLE
MD060 compact conflicts with mdformat empty table cells

### ISSUE THREAD

**Issue by theredspoon** [rvben/rumdl#629]
### Problem or motivation

`rumdl` MD060 with `style = "compact"` conflicts with `mdformat`/`mdformat-gfm` output for empty table cells.

`mdformat` formats an empty table cell as a single visible gap between pipes:

```markdown
| A | B |
| --- | --- |
| x | |
```

`rumdl` MD060 compact reports that table as misaligned and fixes it to two spaces between the final pipes:

```markdown
| A | B |
| --- | --- |
| x |  |
```

Running `mdformat` again changes it back to `| x | |`, so projects that run `mdformat` before `rumdl check` cannot enable MD060 compact when tables contain empty cells.

### Reproduction

With `rumdl 0.1.91` and `mdformat 1.0.0`:

```bash
cat > t.md <<'MD'
# T

| A | B |
| --- | --- |
| x | |
MD

cat > .rumdl.toml <<'TOML'
[MD060]
enabled = true
style = "compact"
TOML

mdformat t.md
rumdl check --no-cache t.md
```

Actual result:

```text
t.md:5:1: [MD060] Table columns should be aligned [*]
```

Then:

```bash
rumdl check --no-cache --fix t.md
cat t.md
mdformat t.md
cat t.md
```

`rumdl` produces:

```markdown
| x |  |
```

`mdformat` returns it to:

```markdown
| x | |
```

### Proposed solution

Treat `| |` and `|  |` as equivalent for empty cells when `MD060.style = "compact"`, or otherwise add a compatibility option that allows mdformat-style empty compact cells.

My preference would be for compact mode to accept mdformat's empty-cell representation without requiring another option, because both forms render identically and the current behavior creates a formatter/linter loop.

### Alternatives considered

- Disable MD060: works around the loop, but loses table-format validation.
- Replace empty cells with placeholders such as `N/A` or `-`: avoids the conflict but changes document content and is not always semantically appropriate.
- Ask mdformat for a table-cell spacing option: less likely to fit mdformat's low-configuration formatter model, and this is specifically an MD060 compatibility issue for projects that use rumdl as the linter.

### Feature category

Compatibility / lint rule behavior

### Example markdown

```markdown
| A | B |
| --- | --- |
| x | |
```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

### Additional context

I searched existing rumdl issues/PRs/discussions with `gh`. The closest open issue I found was #589, but that is about markdownlint `aligned_delimiter` compatibility rather than mdformat empty-cell compatibility.

**Comment by rvben** [comment]
Fixed in v0.1.92.

In compact style, rumdl now treats `| |` (single space between pipes) as the canonical empty table cell, matching mdformat's output. The two-space form `|  |` normalizes to single-space, so the previous formatter/linter loop is resolved without any config change.

If you also want rumdl to lay down alongside mdformat on the other syntax overlaps (horizontal rule style, code fence style, list markers), there's a paste-ready preset in the docs: https://github.com/rvben/rumdl/blob/main/docs/mdformat-comparison.md#using-both-tools-together

Thanks for the detailed repro, @theredspoon.

**Comment by theredspoon** [comment]
@rvben Nice! Impressive turnaround, thanks for taking care of it.
